lib/ebaze/auctions.ex

```diff
@@ -8,6 +8,23 @@ defmodule Ebaze.Auctions do
     Repo.all(Auction)
   end
 
+  def list_my_auctions(user) do
+    Repo.all(
+      from auction in Auction,
```

Comment:
Could you refactor this so the logic for listing out all actions is extracted out from the where query by using piping?

Reply:
@ssunday is`from(auction in Auction, where: auction.created_by_id == ^user.id)
    |> Repo.all()` good?

Reply:
Could you extract out `from(c in Auction)` and then pipe the where to it?

Reply:
`Auction
    |> where([auction], auction.created_by_id == ^user.id)
    |> Repo.all()`

Reply:
I think it can be `where(created_by_id: ^user.id)`, but yes!

Reply:
thats super cool!